lib/elixir/lib/gen_server.ex

```diff
@@ -1205,4 +1217,11 @@ defmodule GenServer do
   def whereis({name, node} = server) when is_atom(name) and is_atom(node) do
     server
   end
+
+  @doc false
+  # TODO: support 2nd argument with maximum length and depth of the displayed
+  # message
```

Comment:
Shall we tackle this TODO now? I would like to avoid addnig new TODOs to the code. :)

Reply:
I will try to tackle it before merging.

Reply:
I decided to ignore it as OTP in general ignores it as well.